src/tapis-ui/src/components/files/FileListing/FileListing.tsx

```diff
@@ -6,9 +6,15 @@ import { FileListingCallback, FileListingDirectory } from 'tapis-redux/files/typ
 import { Config, TapisState } from 'tapis-redux/types';
 import { Files } from '@tapis/tapis-typescript';
 import { useSelector } from 'react-redux';
+import { getId } from "tapis-util";
 
 export type OnSelectCallback = (file: Files.FileInfo) => any;
 
+interface IdFile {
+  id: string,
+  file: Files.FileInfo
```

Comment:
What does the id field do when we wrap the file in another object?

Reply:
Ah OK I see what you're trying to do. Generally speaking, for React when it complains about object uniqueness and the source object doesn't have a unique property itself (like system.id) we provide uuiv4 for a key property.

Reply:
I was considering using that but wasn't sure if it was worth including the uuid package for something like that, I'll change this to use uuids.

Reply:
This will be resolved in https://github.com/tapis-project/tapis-ui/pull/86